crates/aranya-client/src/config.rs

```diff
@@ -80,9 +83,67 @@ impl Default for SyncPeerConfigBuilder {
     }
 }
 
+#[derive(Clone)]
+pub struct QuicSyncConfig {
+    seed_mode: SeedMode,
+}
+
+impl QuicSyncConfig {
+    pub fn builder() -> QuicSyncConfigBuilder {
+        QuicSyncConfigBuilder::default()
+    }
+}
+
+#[derive(Clone, Default)]
+pub struct QuicSyncConfigBuilder {
+    seed_mode: SeedMode,
+}
+
+impl QuicSyncConfigBuilder {
+    /// Sets the seed mode to 'Generate'.
+    /// This is the default mode.
+    ///
+    /// This option is only valid when used in [`super::Client::create_team`].
+    /// Overwrites [`Self::wrapped_seed`] and [`Self::seed_ikm`].
+    pub fn gen_seed(mut self) -> Self {
+        self.seed_mode = SeedMode::Generate;
+        self
+    }
+
+    /// Sets the seed mode to 'IKM'.
+    ///
+    /// This option is valid in both [`super::Client::create_team`] and [`super::Team::add_team`].
+    /// Overwrites [`Self::wrapped_seed`] and [`Self::gen_seed`]
+    pub fn seed_ikm(mut self, ikm: [u8; 32]) -> Self {
```

Comment:
@jdygert-spok, @ssweetney-spok we should define a constant somewhere for the seed size. Or use an existing constant if it's already defined somewhere else:
```suggestion
    pub fn seed_ikm(mut self, ikm: [u8; SEED_SIZE]) -> Self {
```